pkgs/tools/inputmethods/libinput-gestures/default.nix

```diff
@@ -0,0 +1,35 @@
+{lib, stdenv, fetchFromGitHub, bash, fakeroot, libinput, xdotool, makeWrapper}:
+stdenv.mkDerivation rec {
+  pname = "libinput-gestures";
+  version = "2.32";
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "bulletmark";
+    repo = "libinput-gestures";
+    rev = version;
+    sha256 = "1by6sabx0s8sd9w5675gc26q7yccxnxxsjg4dqlb6nbs0vcg81s7";
+  };
+
+  buildInputs = [ bash fakeroot makeWrapper ];
+
+  installPhase =
+    ''
+      mkdir -p $out
+      fakeroot -- bash libinput-gestures-setup -d "$out" install
+      mv $out/usr/bin $out/bin-unwrapped
+      mv $out/usr/share $out/share
+      rmdir $out/usr
+      mkdir $out/bin
+      sed -i "s,/usr/bin/,,g" $out/share/applications/*.desktop
```

Comment:
1. Very minor nitpick - you might as well use ```substituteInPlace``` which handles the escaping for you. 
2. If you're replacing the ```Exec=``` line, please provide the full path to the binary in ```/nix/store```

Reply:
Regarding 2: I was unsure about this, but Firefox and others seemed to just provide the name. Fixing it now.